packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -4254,6 +4258,36 @@ module.exports = function(config, DB) {
             }
           );
       });
+
+      it('should create disabled key and then verify it', async () => {
```

Comment:
No test for the negative case? e.g. that there is no user in the db and the stored proc errors correctly and its caught correctly.

Reply:
Added! I will add some more negative tests in auth-server as follow up.